src/redux/categories/categories.js

```diff
@@ -0,0 +1,23 @@
+// Actions
+const STATUS_COMPLETED = 'STATUS_COMPLETED';
+const STATUS_IN_PROGRESS = 'STATUS_IN_PROGRESS';
+
+// Initial State
+const initialState = { categories: [] };
+
+// Action creator
+export const completedStatusAction = () => ({ type: STATUS_COMPLETED });
+export const inProgressStatusAction = () => ({ type: STATUS_IN_PROGRESS });
+
+// Reducer
+const categoriesReducer = (state = initialState, action) => {
+  if (action.type === STATUS_COMPLETED || action.type === STATUS_IN_PROGRESS) {
+    return {
+      ...state,
+      categories: [...state.categories, 'Under Construction'],
+    };
+  }
+  return state;
```

Comment:
- [ ] as part of the requirements, it should be only one action type for categories which is `CHECK_STATUS`, so kindly remove the `STATUS_IN_PROGRESS` & `STATUS_COMPLETED` and add only  `CHECK_STATUS`, and please don't forget to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) in here as well, so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS` , kindly fix it 🙏